docs/quickstart.rst

```diff
@@ -33,67 +49,47 @@ On Fedora 23 and 24, this works:
 For other operating systems, just do some web searches for "ffi.h" with the name of your OS.
 
 
-Python 3
-^^^^^^^^
+Dependency 2: Python 3.5+
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The BigchainDB Python Driver uses Python 3, preferably a recent version. You can check your version of Python using:
+The BigchainDB Python Driver uses Python 3.5+. You can check your version of Python using:
 
 .. code-block:: bash
 
     python --version
 
-If it says your version is 2.x, try:
-
-.. code-block:: bash
-
-    python3 --version
-
 An easy way to install a specific version of Python, and to switch between versions of Python, is to use `virtualenv <https://virtualenv.pypa.io/en/latest/>`_. Another option is `conda <http://conda.pydata.org/docs/>`_.
 
 
-pip
-^^^
+Dependency 3: pip
+^^^^^^^^^^^^^^^^^
 
-You also need to get a recent Python 3 version of ``pip``, the Python package manager. It might be called ``pip3`` on your OS.
+You also need to get a recent, Python 3 version of ``pip``, the Python package manager.
 
 If you're using virtualenv or conda, then each virtual environment should include an appropriate version of ``pip``.
 
 You can check your version of ``pip`` using:
 
-.. code-block::
+.. code-block:: bash
 
     pip --version
 
-(You might also try the same command with ``pip3`` in place of ``pip``.)
-
 ``pip`` was at version 9.0.0 as of November 2016. If you need to upgrade your version of ``pip``, see the page about that in the `BigchainDB Server docs <https://docs.bigchaindb.com/projects/server/en/latest/appendices/install-latest-pip.html>`_.
 
```

Comment:
I know this wasn't added in this PR, but since we are looking at now, one question:

Wouldn't it be best to point to pip itself for the "how to" things: https://pip.pypa.io/en/stable/installing/ ?